demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -717,22 +891,24 @@ def build_dependency_graph(pack_id, id_set):
         return graph
 
     @staticmethod
-    def find_dependencies(pack_name, id_set_path=None):
+    def find_dependencies(pack_name, id_set_path=None, debug_file_path=None):
         """
         Main function for dependencies search and pack metadata update.
 
         Args:
             pack_name (str): pack id, currently pack folder name is in use.
             id_set_path (str): id set json.
+            debug_file_path (str): path to dependency explanations file.
 
         """
         if not id_set_path:
-            id_set = IDSetCreator(output=None, print_logs=False).create_id_set()
+            id_set = IDSetCreator(print_logs=False).create_id_set()
         else:
             with open(id_set_path, 'r') as id_set_file:
                 id_set = json.load(id_set_file)
-
-        dependency_graph = PackDependencies.build_dependency_graph(pack_id=pack_name, id_set=id_set)
+        with VerboseFile(debug_file_path) as verbose_file:
```

Comment:
What happens if the `debug_file_path` does not exist?

Reply:
the object ignores all the prints, see the implementation at the top of the file